client/Angular/services/gameService.js

```diff
@@ -179,33 +191,32 @@ ClonageApp.service('gameService', ['communicationService', 'dynamicTextService',
 		we will get first when refreshing. The filledInText is another property attached to each answer just on the
 		client side used to display each answer in its context.
 		*/
-		if (answers !== undefined) {
-			answers.forEach(function(answer) {
-				answer.filledInText = dynamicTextService.fillInSelections(currentQuestion.text, answer.answersText);
+		if (roundSubmissionData !== undefined) {
+			roundSubmissionData.forEach(function(submission) {
+				submission.filledInText = dynamicTextService.fillInSelections(currentQuestion.text, submission.submissionsText);
 			});
 		}
 	}
 
-	function _setChosenAnswers(data) {
-		answers = data.answers;
+	function _setRoundSubmissionData(data) {
+
+		roundSubmissionData = data.roundSubmissionData;
+		currentNumberOfSubmissions = data. currentNumberOfSubmissions;
+		currentNumberOfVotes = data.currentNumberOfVotes;
 
+		if(data.dontResetAnswers === undefined){
+			answersToVisualise = data.roundSubmissionData;
+		}
+
+		console.log("currentVOTES" + currentNumberOfVotes);
 		//generating the filled in question based on the question text and submitted answers as above
 		if (currentQuestion !== undefined) {
-			answers.forEach(function(answer) {
-				answer.filledInText = dynamicTextService.fillInSelections(currentQuestion.text, answer.answersText);
+			roundSubmissionData.forEach(function(submission) {
+				submission.filledInText = dynamicTextService.fillInSelections(currentQuestion.text, submission.submissionsText);
 			});
 		}
 
-		countdown = data.countdown;
```

Comment:
countdown is set in the receive question function which is also called on refresh so this shouldn't make a difference to timer behaviour